test/test_ops.py

```diff
@@ -2195,7 +2195,15 @@ def test_fake_crossref_backward_no_amp(self, device, dtype, op):
     @skipOps('TestFakeTensor', 'test_fake_crossref_backward_amp', fake_backward_xfails | fake_autocast_backward_xfails)
     def test_fake_crossref_backward_amp(self, device, dtype, op):
         self._test_fake_crossref_helper(device, dtype, op, torch.cuda.amp.autocast)
-
+    @ops([op for op in op_db if op.is_factory_method], allowed_dtypes=(torch.float,))
+    def test_strided_layout(self, device, dtype, op):
+        samples = op.sample_inputs(device, dtype)
+        for sample in samples:
+            default_result = op(sample.input, *sample.args, **sample.kwargs)
+            self.assertEqual(default_result.layout, torch.strided)
+            strided_result = op(sample.input.to(layout=torch.strided), *sample.args, **sample.kwargs)
+            self.assertEqual(strided_result, default_result)
```

Comment:
You don't want to test this as this is not testing the layout. You want to test that the layout is correct